test/dummies/v1beta2/vars.go

```diff
@@ -130,6 +137,95 @@ func CAPCNetToCSAPINet(net *infrav1.Network) *csapi.Network {
 	}
 }
 
+func setCSMachineOwnerCrd() {
+	CSMachineOwnerCRD = &apiextensionsv1.CustomResourceDefinition{
```

Comment:
I feel like this could have just been imported instead of redefined?

Reply:
As in this could just be a machineset or similar imported from CAPI instead of creating your own.

Reply:
As I mentioned in above response, even machineset, kubeadmControlPlane and etcdadmCluster have similar pattern, like spec.replicas, status.replicas, they also have difference. For example, machineset does not have status.ready and etcdadmcluster does not have status.readyReplicas. kubeadmControlPlane has all fields but they are required fields and could not test `not found` logic.